torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -182,6 +183,9 @@ def __init__(
         super().__init__()
 
         assert len(configs) > 0, "Non-empty TritonConfig list required for compiling"
+        # makes sure there are no pre-hooks on any of the triton configs
+        (validate_triton_config(cfg) for cfg in configs)
```

Comment:
I don't think this will actually do anything since it is a unused generator:
```
>>> (print(x) for x in range(10))
<generator object <genexpr> at 0x7e9bd143e110>
>>> [print(x) for x in range(10)]
0
1
2
3
4
5
6
7
8
9
[None, None, None, None, None, None, None, None, None, None]
>>> 
```

Reply:
thanks for catching!